.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: main
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
this is setup for `main`, but the repo is currently using `master`. you can rename the branch under https://github.com/Ayolland/borksy/settings/branches > "Default Branch > "Rename branch" (borksy isn't currently using the branch name for anything, so this should be safe to change)